src/components/menu/menu.spec.ts

```diff
@@ -42,10 +47,12 @@ import { MenuModule } from './menu.module';
     `
 })
 export class MenuTestComponent {
+    disabled: boolean = false;
     @ViewChild('menuTrigger', { static: true }) trigger: MenuTriggerDirective;
     @ViewChild('subMenuTrigger', { static: true }) subMenuTrigger: MenuTriggerDirective;
 
-    item1Activated(_: MouseEvent | KeyboardEvent): void {}
+    item1Activated(_: MouseEvent | KeyboardEvent): void { }
```

Comment:
Perhaps we could just name these functions `onActivate` and `onClick`, that way if we change the tests in future other items in the test can also use these functions, as they aren't really specific to any item.

Also the type can just be `MouseEvent`, a click event, regardless of if the mouse or keyboard is used, will fire a `MouseEvent`

Reply:
The `activate` event should be `MouseEvent | KeyboardEvent`, this is a custom output which combines click and keydown.